manifest.json

```diff
@@ -669,6 +669,28 @@
               "variant": "v8"
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/windows-nano/1803/helix/amd64",
```

Comment:
The path should be `src/nanoserver/1803/helix/amd64`.  The convention is for the path to match the tag.

Reply:
fixed.